src/compiler/crystal/exception.cr

```diff
@@ -257,7 +257,7 @@ module Crystal
         source_slice = source.lines[from_index...line_number]
         source_slice, spaces_removed = minimize_indentation(source_slice)
 
-        io << Crystal.with_line_numbers(source_slice, line_number, @color, line_number_start = from_index + 1)
+        io << Crystal.with_line_numbers(source_slice, line_number, @color, from_index + 1)
```

Comment:
```suggestion
        io << Crystal.with_line_numbers(source_slice, line_number, @color, line_number_start: from_index + 1)
```

This was probably the intended original source FWIW

Reply:
@bcardiff It doesn't matter since it's the same as positional argument, but I'll change it to be more explicit.

Reply:
It is clearer with `line_number_start` what the argument is. The others have already self-explicit variable names, that's probably why positional arguments were not used for them.

Reply:
Let's leave the named argument, is clearer.